app/addons/setup/setup.js

```diff
@@ -66,6 +99,32 @@ var SetupCurrentAdminPassword = React.createClass({
 
 });
 
+var SetupNodeCountSetting = React.createClass({
+  getInitialState: function () {
+    return {
+      nodeCountValue: this.props.nodeCountValue
+    };
+  },
+
+  handleNodeCountChange: function (event) {
+    this.props.onAlterNodeCount(event);
```

Comment:
Again was just copying this from other functions in this function...will try to change it